scripts/mac-toolchain-build

```diff
@@ -157,6 +163,25 @@ function mac_toolchain_build {
     ${SUDO} make install >>../nasm-2.14.02.log 2>&1 || print_fail_and_exit
     echo -en "${CREL}"
 
+    # ninja
+    cd "${TEMP_DIR}"
+    printf "Building    [%02i/%02i] %s " "7" "${TOTAL}" "ninja 1.9.0"
+    [[ "${SUDO}" != "" ]] && ${SUDO} -v
+    tar -xf ninja-1.9.0.tar.gz >/dev/null 2>&1 || print_fail_and_exit
+    cd ninja-1.9.0 >/dev/null 2>&1 || print_fail_and_exit
+    ./configure.py --bootstrap >../ninja-1.9.0.log 2>&1 || print_fail_and_exit
+    ${SUDO} mv ninja /usr/local/bin >>../ninja-1.9.0.log 2>&1 || print_fail_and_exit
+    echo -en "${CREL}"
+
+    # meson
+    cd "${TEMP_DIR}"
+    printf "Building    [%02i/%02i] %s " "8" "${TOTAL}" "meson 0.51.2"
+    [[ "${SUDO}" != "" ]] && ${SUDO} -v
+    tar -xf meson-0.51.2.tar.gz >/dev/null 2>&1 || print_fail_and_exit
+    cd meson-0.51.2 >/dev/null 2>&1 || print_fail_and_exit
+    ${SUDO} python3 setup.py install >>../meson-0.51.2.log 2>&1 || print_fail_and_exit
+    echo -en "${CREL}"
+
     # done
     printf "Building    [%02i/%02i] complete.\n" "${TOTAL}" "${TOTAL}"
     rm -rf "${TEMP_DIR}"
```

Comment:
I believe we'll need to add `${SUDO} ` here to allow removing meson's temp directory properly.